pkg/image/export/main.go

```diff
@@ -513,7 +513,8 @@ images="rancher-images.tar.gz"
 usage () {
     echo "USAGE: $0 [--image-list rancher-images.txt] [--images rancher-images.tar.gz]"
     echo "  [-l|--image-list path] text file with list of images; one image per line."
-    echo "  [-i|--images path] tar.gz generated by docker save."
+		echo "  [-i|--images path] tar.gz generated by docker save."
+		echo "  [-d|--disable-save] disable docker save and only pull images."
```

Comment:
Generally `-d` is used as debug enable, it's better to not use this short form. Also, for the long form, it's better to use a more descriptive arg. Suggest: `[--disable-docker-save]`.

Reply:
No problem, I removed `-d` and swapped it as requested.